helix-lsp/src/transport.rs

```diff
@@ -222,10 +222,13 @@ impl Transport {
         loop {
             match Self::recv_server_message(&mut server_stdout, &mut recv_buffer).await {
                 Ok(msg) => {
-                    transport
-                        .process_server_message(&client_tx, msg)
-                        .await
-                        .unwrap();
+                    match transport.process_server_message(&client_tx, msg).await {
```

Comment:
Oh, why not do `if let Err(err) = ...` rather than `match`, wouldn't that be cleaner since `Ok(_)` branch is not used?

Reply:
I can do that no problem, I was just essentially looking to match the style of the statement in the middle of these two:
https://github.com/helix-editor/helix/blob/aa70411241b9e12d9ad595dfe524d68ab849747c/helix-lsp/src/transport.rs#L247

But I can easily modify all three, since none of them use the `Ok`.